src/middlewares/validation/validation.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable camelcase */
+/* eslint-disable import/prefer-default-export */
+import { signUpValidationSchema, validateData } from '../../validation/index';
+import Util from '../../utils/response';
+
+const util = new Util();
+
+export const validationForSignUp = (req, res, next) => {
+  const {
+    email, password, firstName, lastName
+  } = req.body;
+  const signupBody = {
+    email, password, firstName, lastName
+  };
+  try {
+    const { error, value } = validateData(signupBody, signUpValidationSchema);
+    const errMessage = error ? error.details[0].message : null;
+    if (error) {
+      util.setError(400, errMessage);
+      return util.send(res);
+    }
+    if (error) {
```

Comment:
What is the difference between this and you have on line 18?

Reply:
That was a mistake